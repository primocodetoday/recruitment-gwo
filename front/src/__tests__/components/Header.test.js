import React from 'react';
import { render } from '@testing-library/react';
import { Header } from 'components';

it('<Header/> render properly', () => {
  const { getByText, container } = render(<Header>Testing</Header>);
  expect(getByText('Testing')).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
