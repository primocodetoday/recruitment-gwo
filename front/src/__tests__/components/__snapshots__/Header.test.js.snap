// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasketHeader/> render properly 1`] = `
<header>
  <h4
    class="mb-4 text-uppercase"
  >
    Testing
  </h4>
</header>
`;

exports[`<Header/> render properly 1`] = `
<header>
  <h4
    class="mb-4 text-uppercase"
  >
    Testing
  </h4>
</header>
`;
